engine/crates/engine-v2/src/operation/prepare.rs

```diff
@@ -82,6 +82,9 @@ impl Operation {
             }
         };
 
+        #[cfg(feature = "qp")]
+        crate::operation::plan::plan(schema, operation.clone()).unwrap();
```

Comment:
Oh, here's the clone. Is this temporary or do we have to clone it?

Reply:
Temporary, I just want to see that QP doesn't fail on tests for now